src/box/lua/upgrade.lua

```diff
@@ -1208,8 +1208,17 @@ local function remove_sql_builtin_functions_from_func()
     end
 end
 
+local function grant_rw_access_on__session_settings_to_role_public()
+    local _session_settings = box.space[box.schema.SESSION_SETTINGS_ID]
+    local _priv = box.space[box.schema.PRIV_ID]
+    log.info("grant read,write access on _session_settings space to public role")
+    _priv:insert({ADMIN, PUBLIC, 'space', _session_settings.id,
+                  box.priv.R + box.priv.W})
+end
+
 local function upgrade_to_2_9_1()
     remove_sql_builtin_functions_from_func()
+    grant_rw_access_on__session_settings_to_role_public()
```

Comment:
This should be part of 2.10 upgrade: 2.9 is current version, so users may have already called this function..

Reply:
@Korablev77 actually, the current version of Tarantool's `HEAD` seems to be 2.9.0:

```bash
./tarantool -v
Tarantool 2.9.0-682-g310082478
Target: Linux-x86_64-Debug
```

Reply:
On master branch:
```
./src/tarantool -v
Tarantool 2.10.0-beta1-317-gc9787d572
Target: Linux-x86_64-Debug
```

Reply:
@Korablev77 weird... I am pulling master, as you can see by the commit hash, perhaps, something is wrong with my fork

```bash
git describe --tags
2.9.0-689-gc9787d572
```

Reply:
Fixed: changed to upgrade to 2.10.0. Sorry for the inconvenience.